CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+# 94.2.0
+
+This minor release introduces new APIs to MDCAppBarViewController and MDCSlider. Additionally, it includes some fixes for Banner constraints.
+
+## API changes
```

Comment:
Please add examples for the new features.

Reply:
Addressed in 7c2a04e9c3f23db39add786cc1eb0d4fbdf028e1.